docs/articles/documentation/test-api/index.md

```diff
@@ -18,3 +18,6 @@ It contains the following subsections
 * [Pausing the Test](pausing-the-test.md)
 * [Handling Native Dialogs](handling-native-dialogs.md)
 * [Working with \<iframes\>](working-with-iframes.md)
+
+> If you use [eslint](http://eslint.org/) in your project, get the [TestCafe plugin](https://www.npmjs.com/package/eslint-plugin-testcafe)
+to enable eslint to report errors in test code.
```

Comment:
> to enable eslint to report errors in test code.

As I understand it's not truth. Eslint reports errors in test code without our plugin. This plugin just add our globals (as `test` and `fixture`) to eslint `globals` setting (https://github.com/miherlosev/eslint-plugin-testcafe/blob/master/index.js). Without the plugin eslint will report an error like `"fixture" is not defined`